bigtable/google/cloud/bigtable/table.py

```diff
@@ -276,6 +288,32 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         # We expect an iterator of `data_messages_v2_pb2.ReadRowsResponse`
         return PartialRowsData(response_iterator)
 
+    def mutate_rows(self, rows):
+        """Mutates multiple rows in bulk.
+
+        The method tries to update all specified rows.
+        If some of the rows weren't updated, it would not remove mutations.
+        They can be applied to the row separately.
+        If row mutations finished successfully, they would be cleaned up.
+
+        :type rows: list
+        :param rows: List or other iterable of :class:`.DirectRow` instances.
+
+        :rtype: list
+        :returns: A list of corresponding to each row statuses.
+        """
+        mutate_rows_request = _mutate_rows_request(self.name, rows)
+        client = self._instance._client
+        responses = client._data_stub.MutateRows(mutate_rows_request)
+
+        responses_statuses = [None for _ in range(len(rows))]
+        for response in responses:
+            for entry in response.entries:
+                responses_statuses[entry.index] = entry.status
+                if entry.status.code == 0:
+                    rows[entry.index].clear()
```

Comment:
'rows' has to be iterable. It is specified in a docstring.
Should I add an explicit data type check? (rows is iterable)